azure-pipelines.yml

```diff
@@ -0,0 +1,399 @@
+          -DnewVersion=$(SONAR_PROJECT_VERSION).$(Build.BuildId)
+          --settings $(mavenSettings.secureFilePath)
+          -DgenerateBackupPoms=false -e
+         javaHomeOption: 'JDKVersion'
+         mavenVersionOption: 'Default'
+         jdkVersionOption: '1.11'
+         mavenOptions: $(MAVEN_OPTS)
+      - task: Maven@3
+        displayName: 'Stage to repox'
+        env:
+          PROJECT_VERSION: $(SONAR_PROJECT_VERSION)
+          BUILD_ID: $(Build.BuildId)
+          ARTIFACTORY_DEPLOY_USERNAME: $(ARTIFACTORY_DEPLOY_USERNAME)
+          ARTIFACTORY_DEPLOY_PASSWORD: $(ARTIFACTORY_DEPLOY_PASSWORD)
+        inputs:
+          goals: 'deploy'
+          options: >-
+            $(commonMavenArguments)
+            --settings $(mavenSettings.secureFilePath)
+            -Pdeploy-sonarsource -e -V
+          publishJUnitResults: false
+          javaHomeOption: 'JDKVersion'
+          jdkVersionOption: '1.11'
+          mavenOptions: $(MAVEN_OPTS)
+      - task: PublishPipelineArtifact@1
+        displayName: 'Publish packages as artifacts'
+        inputs:
+          targetPath: 'DeploymentArtifacts/BuildAgentPayload/$(BuildConfiguration)'
+          artifact: 'scanner-packages'
+      - bash: git checkout .
+        name: revertPomChanges
+        displayName: Revert changes made to pom.xml to not break cache feature
+- template: stage-with-burgr-notifications.yml@commonTemplates
+  parameters:
+    burgrName: 'qa'
+    burgrType: 'qa'
+    stageName: 'qa'
+    stageDisplayName: Run ITs
+    stageDependencies: 
+     - build
+    jobs:
+    - job: its
+      displayName: 'Run its'
+      strategy: 
+        matrix: 
+          vs2015_dev: 
+            imageName: "vs2015-win2012r2"
+            SQ_VERSION: "DEV"
+            MSBUILD_PATH: "C:\\Program Files (x86)\\MSBuild\\14.0\\Bin\\MSBuild.exe"
+          vs2015_latest67: 
+            imageName: "vs2015-win2012r2"
+            SQ_VERSION: "LATEST_RELEASE[6.7]"
+            MSBUILD_PATH: "C:\\Program Files (x86)\\MSBuild\\14.0\\Bin\\MSBuild.exe"
+          vs2015_latest79: 
+            imageName: "vs2015-win2012r2"
+            SQ_VERSION: "LATEST_RELEASE[7.9]"
+            MSBUILD_PATH: "C:\\Program Files (x86)\\MSBuild\\14.0\\Bin\\MSBuild.exe"
+          vs2017_dev: 
+            imageName: "vs2017-win2016"
+            SQ_VERSION: "DEV"
+            MSBUILD_PATH: "C:\\Program Files (x86)\\Microsoft Visual Studio\\2017\\Enterprise\\MSBuild\\15.0\\Bin\\MSBuild.exe"
+          vs2017_latest67: 
+            imageName: "vs2017-win2016"
+            SQ_VERSION: "LATEST_RELEASE[6.7]"
+            MSBUILD_PATH: "C:\\Program Files (x86)\\Microsoft Visual Studio\\2017\\Enterprise\\MSBuild\\15.0\\Bin\\MSBuild.exe"
+          vs2017_latest79: 
+            imageName: "vs2017-win2016"
+            SQ_VERSION: "LATEST_RELEASE[7.9]"
+            MSBUILD_PATH: "C:\\Program Files (x86)\\Microsoft Visual Studio\\2017\\Enterprise\\MSBuild\\15.0\\Bin\\MSBuild.exe"
+        maxParallel: 2
```

Comment:
How long does it take with 2 parallel jobs? (I think we have more agents available if needed).

Reply:
It's taking around 40-45min overall. I'll test with 4.

Reply:
So finally i removed the maxParallel value, job is done in 11min-ish now :)